pallets/connectors/src/lib.rs

```diff
@@ -240,7 +240,7 @@ pub mod pallet {
 		/// The Router for a given domain was set
 		SetDomainRouter {
 			domain: Domain,
-			router: Router<CurrencyIdOf<T>>,
+			router: Box<Router<CurrencyIdOf<T>>>,
```

Comment:
Minor comment:
I guess this is because of `clippy` complaints about this. Could it maybe be better to add the `Box` into the `Router` enum, nearest to the big variant that causes this? This way, if tomorrow we have a simpler router, we do not need to allocate it.

Reply:
Good one, will do ✌🏻

Reply:
Done, it was the `VersionedMultiLocation` that needed to be boxed 👍 thanks again